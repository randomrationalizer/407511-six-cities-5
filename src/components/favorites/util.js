export const getCitiesFromOffers = (offers) => {
  const cities = offers.slice().map((offer) => offer.city.name).sort();
  return [...new Set(cities)];
};
