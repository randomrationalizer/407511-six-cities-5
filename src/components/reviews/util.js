const REVIEW_MIN_LENGTH = 50;
const REVIEW_MAX_LENGTH = 300;
const RATING_MIN = 1;
const RATING_MAX = 5;

export const checkCommentValidity = (comment) => {
  return comment.length >= REVIEW_MIN_LENGTH && comment.length <= REVIEW_MAX_LENGTH ? true : false;
};

export const checkRatingValidity = (rating) => {
  return rating >= RATING_MIN && rating <= RATING_MAX ? true : false;
};
