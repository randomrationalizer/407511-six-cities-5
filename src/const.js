export const cities = [
  {
    name: `Paris`,
    coords: {
      latitude: 48.864716,
      longitude: 2.349014
    }
  },
  {
    name: `Cologne`,
    coords: {
      latitude: 50.935173,
      longitude: 6.953101
    }
  },
  {
    name: `Brussels`,
    coords: {
      latitude: 50.8505,
      longitude: 4.3488
    }
  },
  {
    name: `Amsterdam`,
    coords: {
      latitude: 52.38333,
      longitude: 4.9
    }
  },
  {
    name: `Hamburg`,
    coords: {
      latitude: 53.551086,
      longitude: 9.993682
    }
  },
  {
    name: `Dusseldorf`,
    coords: {
      latitude: 51.233334,
      longitude: 6.783333
    }
  }
];

export const OfferType = {
  MAIN: `cities`,
  NEARBY: `near-places`,
  FAVORITES: `favorites`
};

export const MapType = {
  MAIN: `cities`,
  PROPERTY: `property`
};

export const SortType = {
  DEFAULT: `DEFAULT`,
  PRICE_UP: `PRICE_UP`,
  PRICE_DOWN: `PRICE_DOWN`,
  RATING_DOWN: `RATING_DOWN`
};
