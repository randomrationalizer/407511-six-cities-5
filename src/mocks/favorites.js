export const favorites = [`1`, `3`, `4`];
