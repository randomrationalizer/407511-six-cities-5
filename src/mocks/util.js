import configureStore from "redux-mock-store";

export const noop = () => {};

export const getMockStore = (store) => {
  const storeConfig = configureStore();
  return storeConfig(store);
};

export const nope = () => {
  return Promise.resolve({});
};
