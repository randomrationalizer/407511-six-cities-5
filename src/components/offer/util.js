import dayjs from "dayjs";
import {OfferType} from "../../const";


export const capitalize = (str) => {
  return str.charAt(0).toUpperCase() + str.slice(1);
};

// Возвращает дату в виде строки в формате: 2019-04-24
export const formatDate = (date) => {
  return dayjs(date).format(`YYYY-MM-DD`);
};

// Возвращает дату в виде строки в формате: January 2020
export const humanizeDate = (date) => {
  return dayjs(date).format(`MMMM YYYY`);
};

// Сравнение для сортировки отзывов в хронологическом порядке
export const sortByDate = (a, b) => {
  return dayjs(a.date) - dayjs(b.date);
};

export const getDescriptionSentences = (text) => {
  let description = text.split(`. `);
  description[description.length - 1] = description[description.length - 1].replace(`.`, ``);

  return description;
};

export const isFavoritesCard = (type) => {
  return type === OfferType.FAVORITES;
};

export const isMainPageCard = (type) => {
  return type === OfferType.MAIN;
};
