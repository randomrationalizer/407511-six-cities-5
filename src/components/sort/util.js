import {SortType} from "../../const";

export const sortTypeToTitle = {
  [SortType.DEFAULT]: `Popular`,
  [SortType.PRICE_UP]: `Price: low to high`,
  [SortType.PRICE_DOWN]: `Price: high to low`,
  [SortType.RATING_DOWN]: `Top rated first`
};
