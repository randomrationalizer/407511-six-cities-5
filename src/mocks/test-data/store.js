import configureStore from "redux-mock-store";
import {NameSpace} from "../../store/root-reducer";
import mockOffers from "./offers";
import mockReviews from "./reviews";
import mockNearbyOffers from "./nearby-offers";
import mockCurrentOffer from "./current-offer";
import mockUserInfo from "./user-info";
import mockFavorites from "./favorites";
import mockCities from "./cities";
import {SortType, AuthorizationStatus, DEFAULT_CITY} from "../../const";


export default {
  [NameSpace.APP_DATA]: {
    currentCity: DEFAULT_CITY,
    allOffers: mockOffers,
    cities: mockCities,
    currentSort: SortType.DEFAULT
  },
  [NameSpace.USER]: {
    authorizationStatus: AuthorizationStatus.AUTH,
    isAuthRequestComplete: true,
    userInfo: mockUserInfo,
    favorites: mockFavorites
  },
  [NameSpace.CURRENT_OFFER]: {
    offer: mockCurrentOffer,
    reviews: mockReviews,
    nearbyOffers: mockNearbyOffers
  },
  [NameSpace.LOAD_STATUS]: {
    isOffersLoaded: true,
    isFavoritesLoaded: true,
    isCurrentOfferLoaded: true,
    errorMessage: null,
    isLoadFinished: true
  }
};
