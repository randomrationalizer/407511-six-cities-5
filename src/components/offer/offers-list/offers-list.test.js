import React from "react";
import {Provider} from 'react-redux';
import {BrowserRouter} from "react-router-dom";
import renderer from "react-test-renderer";
import OffersList from "./offers-list";
import store from "../../../mocks/test-data/store";
import mockNearbyOffers from "../../../mocks/test-data/nearby-offers";
import {getMockStore, noop} from "../../../mocks/util";
import {OfferType} from "../../../const";


const mockStore = getMockStore(store);


describe(`Should OffersList component renders correctly`, () => {
  it(`for main page`, () => {
    const tree = renderer
      .create(
          <Provider store={mockStore}>
            <BrowserRouter>
              <OffersList
                offers={mockNearbyOffers}
                onOfferHover={noop}
                offerType={OfferType.MAIN}
              />
            </BrowserRouter>
          </Provider>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`for property page`, () => {
    const tree = renderer
      .create(
          <Provider store={mockStore}>
            <BrowserRouter>
              <OffersList
                offers={mockNearbyOffers}
                onOfferHover={noop}
                offerType={OfferType.NEARBY}
              />
            </BrowserRouter>
          </Provider>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`for favorites page`, () => {
    const tree = renderer
      .create(
          <Provider store={mockStore}>
            <BrowserRouter>
              <OffersList
                offers={mockNearbyOffers}
                onOfferHover={noop}
                offerType={OfferType.FAVORITES}
              />
            </BrowserRouter>
          </Provider>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
