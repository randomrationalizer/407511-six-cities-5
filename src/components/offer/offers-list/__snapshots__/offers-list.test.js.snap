// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OffersList component renders correctly for favorites page 1`] = `
Array [
  <article
    className="favorites__card place-card card-favorites"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="place-card__mark"
    >
      <span>
        Premium
      </span>
    </div>
    <div
      className="favorites__image-wrapper place-card__image-wrapper"
    >
      <a
        href="/hotels/42"
        onClick={[Function]}
      >
        <img
          alt="Amazing and Extremely Central Flat"
          className="place-card__image"
          height={110}
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg"
          width={150}
        />
      </a>
    </div>
    <div
      className="favorites__card-info place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            275
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button  button"
          onClick={[Function]}
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height={19}
            width={18}
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "68%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="/hotels/42"
          onClick={[Function]}
        >
          Amazing and Extremely Central Flat
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        House
      </p>
    </div>
  </article>,
  <article
    className="favorites__card place-card card-favorites"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="favorites__image-wrapper place-card__image-wrapper"
    >
      <a
        href="/hotels/84"
        onClick={[Function]}
      >
        <img
          alt="Nice, cozy, warm big bed apartment"
          className="place-card__image"
          height={110}
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg"
          width={150}
        />
      </a>
    </div>
    <div
      className="favorites__card-info place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            647
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button  button"
          onClick={[Function]}
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height={19}
            width={18}
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "58%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="/hotels/84"
          onClick={[Function]}
        >
          Nice, cozy, warm big bed apartment
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        House
      </p>
    </div>
  </article>,
  <article
    className="favorites__card place-card card-favorites"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="favorites__image-wrapper place-card__image-wrapper"
    >
      <a
        href="/hotels/44"
        onClick={[Function]}
      >
        <img
          alt="House in countryside"
          className="place-card__image"
          height={110}
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg"
          width={150}
        />
      </a>
    </div>
    <div
      className="favorites__card-info place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            173
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button  button"
          onClick={[Function]}
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height={19}
            width={18}
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "90%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="/hotels/44"
          onClick={[Function]}
        >
          House in countryside
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        Room
      </p>
    </div>
  </article>,
]
`;

exports[`Should OffersList component renders correctly for main page 1`] = `
Array [
  <article
    className="cities__place-card place-card card-main"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="place-card__mark"
    >
      <span>
        Premium
      </span>
    </div>
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="/hotels/42"
        onClick={[Function]}
      >
        <img
          alt="Amazing and Extremely Central Flat"
          className="place-card__image"
          height={200}
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg"
          width={260}
        />
      </a>
    </div>
    <div
      className=" place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            275
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button  button"
          onClick={[Function]}
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height={19}
            width={18}
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "68%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="/hotels/42"
          onClick={[Function]}
        >
          Amazing and Extremely Central Flat
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        House
      </p>
    </div>
  </article>,
  <article
    className="cities__place-card place-card card-main"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="/hotels/84"
        onClick={[Function]}
      >
        <img
          alt="Nice, cozy, warm big bed apartment"
          className="place-card__image"
          height={200}
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg"
          width={260}
        />
      </a>
    </div>
    <div
      className=" place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            647
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button  button"
          onClick={[Function]}
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height={19}
            width={18}
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "58%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="/hotels/84"
          onClick={[Function]}
        >
          Nice, cozy, warm big bed apartment
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        House
      </p>
    </div>
  </article>,
  <article
    className="cities__place-card place-card card-main"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="/hotels/44"
        onClick={[Function]}
      >
        <img
          alt="House in countryside"
          className="place-card__image"
          height={200}
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg"
          width={260}
        />
      </a>
    </div>
    <div
      className=" place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            173
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button  button"
          onClick={[Function]}
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height={19}
            width={18}
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "90%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="/hotels/44"
          onClick={[Function]}
        >
          House in countryside
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        Room
      </p>
    </div>
  </article>,
]
`;

exports[`Should OffersList component renders correctly for property page 1`] = `
Array [
  <article
    className="near-places__card place-card card-nearby"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="place-card__mark"
    >
      <span>
        Premium
      </span>
    </div>
    <div
      className="near-places__image-wrapper place-card__image-wrapper"
    >
      <a
        href="/hotels/42"
        onClick={[Function]}
      >
        <img
          alt="Amazing and Extremely Central Flat"
          className="place-card__image"
          height={200}
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg"
          width={260}
        />
      </a>
    </div>
    <div
      className=" place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            275
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button  button"
          onClick={[Function]}
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height={19}
            width={18}
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "68%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="/hotels/42"
          onClick={[Function]}
        >
          Amazing and Extremely Central Flat
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        House
      </p>
    </div>
  </article>,
  <article
    className="near-places__card place-card card-nearby"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="near-places__image-wrapper place-card__image-wrapper"
    >
      <a
        href="/hotels/84"
        onClick={[Function]}
      >
        <img
          alt="Nice, cozy, warm big bed apartment"
          className="place-card__image"
          height={200}
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg"
          width={260}
        />
      </a>
    </div>
    <div
      className=" place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            647
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button  button"
          onClick={[Function]}
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height={19}
            width={18}
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "58%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="/hotels/84"
          onClick={[Function]}
        >
          Nice, cozy, warm big bed apartment
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        House
      </p>
    </div>
  </article>,
  <article
    className="near-places__card place-card card-nearby"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="near-places__image-wrapper place-card__image-wrapper"
    >
      <a
        href="/hotels/44"
        onClick={[Function]}
      >
        <img
          alt="House in countryside"
          className="place-card__image"
          height={200}
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg"
          width={260}
        />
      </a>
    </div>
    <div
      className=" place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            173
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button  button"
          onClick={[Function]}
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height={19}
            width={18}
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "90%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="/hotels/44"
          onClick={[Function]}
        >
          House in countryside
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        Room
      </p>
    </div>
  </article>,
]
`;
